pages/aboutscript.js

```diff
@@ -0,0 +1,12 @@
+const HamLine1 = document.querySelector('.HamLine1');
+const HamLine2 = document.querySelector('.HamLine2');
+const HamLine3 = document.querySelector('.HamLine3');
+const mobileMenu = document.querySelector('.MobileMenu');
+const hamburgerButton = document.querySelector('.hamburger');
+
+hamburgerButton.addEventListener('click', () => {
+  mobileMenu.classList.toggle('hidden');
```

Comment:
- [ ] Please kindly make sure the home page link is present in the mobile menu links, Kindly fix that as well 🙃